### PR TITLE
Create Subscription Modal

### PR DESCRIPTION
**Pull request by AFahgan**
image

### CODE REVIEW

File: client/src/Components/SingleGym/Modal/index.jsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import {
+  FormControl,
+  InputLabel,
+  MenuItem,
+  OutlinedInput,
+  Select,
+} from "@mui/material";
+
+function Modal() {
+  const subscriptionSchema = Yup.object().shape({
```
Comment (AhmedQeshta) [inline client/src/Components/SingleGym/Modal/index.jsx:21]: this ` subscriptionSchema `, can put it  outside the component `Modal`

**Review by AhmedQeshta — CHANGES_REQUESTED** [review]

**Review by AhmedQeshta — APPROVED** [review]

File: client/src/Components/SingleGym/Modal/index.jsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  FormControl,
+  InputLabel,
+  MenuItem,
+  OutlinedInput,
+  Select,
+} from "@mui/material";
+
+const subscriptionSchema = Yup.object().shape({
+  username: Yup.string().required("حقل اسم المستخدم مطلوب"),
+  phone: Yup.string()
```
Comment (LinaYahya) [inline client/src/Components/SingleGym/Modal/index.jsx:22]:
```suggestion
  userPhone: Yup.string()
```
as it stored in the database

File: client/src/Components/SingleGym/Modal/index.jsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    .length(10, "رقم الهاتف غير صحيح")
+    .required("حقل رقم الهاتف مطلوب"),
+  subscription: Yup.string().required("حقل الاشتراك مطلوب"),
```
Comment (LinaYahya) [inline client/src/Components/SingleGym/Modal/index.jsx:25]:
```suggestion
  type: Yup.string().required("حقل الاشتراك مطلوب"),
```

File: client/src/Components/SingleGym/Modal/index.jsx [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+              >
+                <MenuItem value="month">اشتراك شهري</MenuItem>
+                <MenuItem value="sixMonth">اشتراك ست أشهر</MenuItem>
+              </Select>
+            </FormControl>
+          </DialogContent>
+          <DialogActions>
+            <Button
+              xs={{ fontWeight: "bold" }}
+              onClick={handleClose}
```
Comment (LinaYahya) [inline client/src/Components/SingleGym/Modal/index.jsx:111]:
```suggestion
```
this is the form submit button, so on submit will control this button action

**Review by LinaYahya — CHANGES_REQUESTED** [review]
Good Job Ayman, try to complete it to send the request completely,
And also we need to display six and one month prices in the gum card itself, and in the modal

**Review by AhmedQeshta — APPROVED** [review]
Great work Ayman!

**Review by rawaneyad — APPROVED** [review]
Great Job Ayman 👍🏼

**Review by LinaYahya — APPROVED** [review]

**Pull request merged**